pkg/manifestgen/sourcesecret/sourcesecret.go

```diff
@@ -65,11 +65,9 @@ func Generate(options Options) (*manifestgen.Manifest, error) {
 	var keypair *ssh.KeyPair
 	switch {
 	case options.Username != "" && options.Password != "":
-		// noop
-	case len(options.PrivateKeyPath) > 0:
-		if keypair, err = loadKeyPair(options.PrivateKeyPath, options.Password); err != nil {
-			return nil, err
-		}
+	// noop
```

Comment:
Without indentation it comes across as it relates to the next case, instead of the previous.